pokedex/data/csv/items.csv

```diff
@@ -1085,3 +1085,524 @@ id,identifier,category_id,cost,fling_power,fling_effect_id
+1201,tr13,,2000,10,
+1202,tr14,,2000,10,
+1203,tr15,,16000,110,
+1204,tr16,,6000,80,
+1205,tr17,,4000,10,
+1206,tr18,,6000,80,
+1207,tr19,,4000,80,
+1208,tr20,,6000,10,
+1209,tr21,,4000,10,
+1210,tr22,,10000,90,
+1211,tr23,,4000,10,
+1212,tr24,,16000,120,
+1213,tr25,,6000,80,
+1214,tr26,,2000,10,
+1215,tr27,,4000,10,
+1216,tr28,,16000,120,
+1217,tr29,,4000,10,
+1218,tr30,,4000,10,
+1219,tr31,,10000,100,
+1220,tr32,,6000,80,
+1221,tr33,,6000,80,
+1222,tr34,,6000,120,
+1223,tr35,,6000,90,
+1224,tr36,,10000,95,
+1225,tr37,,4000,10,
+1226,tr38,,4000,10,
+1227,tr39,,16000,120,
+1228,tr40,,2000,10,
+1229,tr41,,6000,85,
+1230,tr42,,10000,90,
+1231,tr43,,16000,130,
+1232,tr44,,4000,10,
+1233,tr45,,10000,90,
+1234,tr46,,4000,10,
+1235,tr47,,6000,80,
+1236,tr48,,4000,10,
+1237,tr49,,4000,10,
+1238,tr50,,10000,90,
+1239,tr51,,4000,10,
+1240,tr52,,6000,10,
+1241,tr53,,16000,120,
+1242,tr54,,4000,10,
+1243,tr55,,16000,120,
+1244,tr56,,6000,80,
+1245,tr57,,6000,80,
+1246,tr58,,6000,80,
+1247,tr59,,6000,80,
+1248,tr60,,6000,80,
+1249,tr61,,10000,90,
+1250,tr62,,6000,85,
+1251,tr63,,6000,80,
+1252,tr64,,16000,120,
+1253,tr65,,10000,90,
+1254,tr66,,16000,120,
+1255,tr67,,10000,90,
+1256,tr68,,4000,10,
+1257,tr69,,6000,80,
+1258,tr70,,10000,80,
+1259,tr71,,16000,130,
+1260,tr72,,16000,120,
+1261,tr73,,16000,120,
+1262,tr74,,10000,80,
+1263,tr75,,16000,100,
+1264,tr76,,6000,10,
+1265,tr77,,6000,10,
+1266,tr78,,10000,95,
+1267,tr79,,6000,10,
+1268,tr80,,6000,10,
+1269,tr81,,6000,95,
+1270,tr82,,4000,20,
+1271,tr83,,4000,10,
+1272,tr84,,6000,80,
+1273,tr85,,2000,10,
+1274,tr86,,10000,90,
+1275,tr87,,6000,80,
+1276,tr88,,6000,10,
+1277,tr89,,16000,110,
+1278,tr90,,10000,90,
+1279,tr91,,4000,10,
+1280,tr92,,6000,80,
+1281,tr93,,10000,85,
+1282,tr94,,10000,95,
+1283,tr95,,6000,80,
+1284,tr96,,10000,90,
+1285,tr97,,10000,85,
+1286,tr98,,6000,85,
+1287,tr99,,6000,80,
+1288,tm00,,10000,,
+1289,lonely-mint,,20,10,
+1290,adamant-mint,,20,10,
+1291,naughty-mint,,20,10,
+1292,brave-mint,,20,10,
+1293,bold-mint,,20,10,
+1294,impish-mint,,20,10,
+1295,lax-mint,,20,10,
+1296,relaxed-mint,,20,10,
+1297,modest-mint,,20,10,
+1298,mild-mint,,20,10,
+1299,rash-mint,,20,10,
+1300,quiet-mint,,20,10,
+1301,calm-mint,,20,10,
+1302,gentle-mint,,20,10,
+1303,careful-mint,,20,10,
+1304,sassy-mint,,20,10,
+1305,timid-mint,,20,10,
+1306,hasty-mint,,20,10,
+1307,jolly-mint,,20,10,
+1308,naive-mint,,20,10,
+1309,serious-mint,,20,10,
+1310,wishing-piece,,20,50,
+1311,cracked-pot,,1600,80,
+1312,chipped-pot,,38000,80,
+1313,hi-tech-earbuds,,0,,
+1314,fruit-bunch,,2200,10,
+1315,moomoo-cheese,,2200,20,
+1316,spice-mix,,400,50,
+1317,fresh-cream,,950,10,
+1318,packaged-curry,,950,10,
+1319,coconut-milk,,950,10,
+1320,instant-noodles,,150,10,
+1321,precooked-burger,,150,10,
+1322,gigantamix,,15000,50,
+1323,wishing-chip,,0,,
+1324,rotom-bike,,0,,
+1325,catching-charm,,0,,
+1326,old-letter,,0,,
+1327,band-autograph,,0,,
+1328,sonias-book,,0,,
+1329,rotom-catalog,,0,,
+1330,★and458,,20,,
```

Comment:
what are these ★ items? unused?

if they're junk, please delete. if not, identifiers should be ascii only.

Reply:
They are [Dynamax Crystals](https://bulbapedia.bulbagarden.net/wiki/Dynamax_Crystal), there are 300 of them, and they are distributed as events. So far only 6 of them have been made available. I replaced the star with a `dynamax-crystal-` prefix